frontend/components/ClubEditPage/ClubEditCard.tsx

```diff
@@ -151,6 +151,48 @@ const Card = ({
     </div>
   )
 }
+interface EmailModalProps {
+  closeModal: () => void
+  email: string
+  setEmail: (inp: string) => void
+  confirmSubmission: () => void
+}
+
+const EmailModal = ({
+  closeModal,
+  email,
+  setEmail,
+  confirmSubmission,
+}: EmailModalProps): ReactElement => {
+  return (
+    <Modal
```

Comment:
Nit: bottom margin is a little large (nonuniform compared to the top margin for the submit button)

Reply:
Done!